lib/ui/common/modal_navigator.dart

```diff
@@ -1,25 +1,27 @@
 import 'package:flutter/material.dart';
 
-class ModalNavigator extends StatelessWidget {
-  ModalNavigator({
+class NavigationScope extends StatelessWidget {
+  NavigationScope({
```

Comment:
I decided to rename as it didn't have anything to do with Modal.. if just wrapped the widget in a Navigator....